dev.py

```diff
@@ -252,13 +252,13 @@ def build() -> None:
         pipargs.append('--user')
     subprocess.check_call(pipargs)
     print('>>>> Installing node modules')
-    subprocess.check_call(['npm', 'install'], shell=True)
```

Comment:
This breaks windows, as npm does not work without shell=True.
I guess we need a platform check.